.circleci/.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will publish npm package on release
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
```

Comment:
@stepancar can we have it publish when we merge to main branch? I feel like it's a lot of bureaucracy to have to create a release after merging.

Reply:
@antoineMoPa , let's skip it for now? I tried to make it similar to what we have for framefusion

Reply:
ok, makes sense to keep similar and eventually move everything at once to a new workflow.